tests/test_schema/customschema/1.1.0/defs.json

```diff
@@ -0,0 +1,315 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "1.1.0/defs.json",
```

Comment:
I haven't looked at this too deeply, but is this what the `v1.1.0` schema def will actually be? Or are these just examples for the sake of demoing an alternative schema?

If these are actually a new schema can we add these in a separate PR that would go in first? Otherwise it seems like we have two important changes going into 1 squashed PR.

Reply:
this is just for testing purposes. In this case, the easiest change to the schema is to just change the version and watch it break.

Reply:
e.g. the point is to check the behavior of `pyhf.schema()` is working as expected, by the fact that we're able to get a previously working spec to not validate when we change the schema.